linera-core/src/chain_worker/state/attempted_changes.rs

```diff
@@ -298,33 +297,33 @@ where
         );
 
         let required_blob_ids = executed_block.required_blob_ids();
-        // Verify that no unrelated blobs were provided.
-        self.state
-            .check_for_unneeded_blobs(&required_blob_ids, blobs)?;
```

Comment:
How do you plan on making sure that only _needed_ blobs are uploaded?

Reply:
The validator doesn't accept blobs for which there is no blob state. Only blobs that have been seen as part of a confirmed block certificate have a blob state.